BREAKING.md

```diff
@@ -19,10 +19,15 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 
 ## 2.0.0-internal.3.0.0 Upcoming changes
 - [Deprecated IPendingFlush](#Deprecated-IPendingFlush)
+- ["connected" event](#Connected-event)
```

Comment:
This should go in the Breaking section not Upcoming.  <version number> Upcoming changes section is for advance notice for breaking changes that will land in the future, that can be dealt with as of <version number>.  e.g. if you add an optional parameter in 4.0 that will be made required in 5.0 you put the info in the "4.0 Upcoming changes" section.